tests/test_traceview.py

```diff
@@ -19,6 +19,9 @@
 
 TV_API_KEY = os.environ.get("TV_API_KEY", None)
 
+# The total_requests API requires a non-default app to measure
+TV_REQUESTS_APP = os.environ.get("TV_REQUESTS_APP", None)
```

Comment:
good idea, i like it!  will have a PR w/fixes shortly :)

On Thu, Oct 15, 2015 at 12:56 PM, Dan Riti <notifications@github.com> wrote:

> In tests/test_traceview.py
> <https://github.com/danriti/python-traceview/pull/14#discussion_r42150207>
> :
>
> > @@ -19,6 +19,9 @@
> >
> >  TV_API_KEY = os.environ.get("TV_API_KEY", None)
> >
> > +# The total_requests API requires a non-default app to measure
> > +TV_REQUESTS_APP = os.environ.get("TV_REQUESTS_APP", None)
>
> @mcodd <https://github.com/mcodd> after thinking about this a bit more,
> what are your thoughts on renaming this to TV_APP_NAME? i was thinking
> that it should be usable by other tests in the future, so encourage reuse
> by naming it something a bit more accessible and less restricted?
>
> thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/danriti/python-traceview/pull/14/files#r42150207>.
>